app/uk/gov/hmrc/individualsemploymentsapi/cache/v2/CacheRepositoryConfiguration.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.individualsemploymentsapi.cache.v2
+
+import play.api.Configuration
+import uk.gov.hmrc.individualsemploymentsapi.cache.{CacheRepositoryConfiguration => BaseConig}
+
+import javax.inject.Inject
+
+class CacheRepositoryConfiguration @Inject()(configuration: Configuration) extends BaseConig {
```

Comment:
Can I just check with you that this is for V2 only? And that V1 still caches in its original mongo instance and repo name?

Reply:
@ajchilts From what i've been able to tell its in the same table and section as the original. I've ran regression before (master) and after (this branch) and from what I can see it creates the same collections and columns. The locations are decided from config which is done by the individual config sections.